cloudformation/cloudformation.yaml

```diff
@@ -469,7 +469,22 @@ Resources:
             "minDaysInAdvance" : 8
             }
           RoleArn: !GetAtt [ fulfilmentTriggerRole, Arn ]
-
+  WeeklyScheduledUploadRule:
+    Type: "AWS::Events::Rule"
+    Properties:
+      Description: "TriggerWeeklyFulfilmentUpload"
+      ScheduleExpression: "cron(00 3 ? * THU *)"
```

Comment:
we could also schedule the upload to happen much later so we have time to react if the fulfilment generation failed on thursday. Otherwise it will upload files generated the day before